discordrb.json

```diff
@@ -57,6 +57,28 @@
             "voiceReceive": "no",
             "voiceSend": "yes",
             "voiceStateUpdate": "yes"
+        },
+        "misc": {
+            "userLoginWithPassword": "yes",
+            "userLoginWithToken": "yes",
+            "botLogin": "yes",
+            "cachesTokens": "yes",
+            "apiDocs": "automatic",
+            "apiDocsUrl": "http://www.rubydoc.info/gems/discordrb",
+            "tutorials": "yes",
```

Comment:
Whether this lib has tutorials available, i. e. pages that describe a particular feature each and how to use it. Usually accompanied by example snippets, but not full runnable code.